src/web/app/pages-session/session-submission-page/session-submission-page.component.spec.ts

```diff
@@ -1,24 +1,616 @@
+      },
+      isEditing: false,
+    },
+  };
+
+  const testRecipientSubmissionForm4: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-4',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answer: 'barry-harris-answer',
+      questionType: FeedbackQuestionType.MCQ,
+    } as FeedbackMcqResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm5: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-5',
+    recipientIdentifier: 'gene-harris-id',
+    responseDetails: {
+      answer: 'gene-harris-answer',
+      questionType: FeedbackQuestionType.MCQ,
+    } as FeedbackMcqResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm6: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-6',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answers: ['answer 1', 'answer 2'],
+      isOther: false,
+      otherFieldContent: 'other field content',
+      questionType: FeedbackQuestionType.MSQ,
+    } as FeedbackMsqResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm7: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-7',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answer: 5,
+    } as FeedbackNumericalScaleResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm8: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-8',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answers: [7, 13],
+    } as FeedbackConstantSumResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm9: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-9',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answer: 20,
+    } as FeedbackContributionResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm10: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-10',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answer: [3, 4],
+    } as FeedbackRubricResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm11: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-11',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      answers: [2, 1],
+    } as FeedbackRankOptionsResponseDetails,
+    isValid: true,
+  };
+
+  const testRecipientSubmissionForm12: FeedbackResponseRecipientSubmissionFormModel = {
+    responseId: 'response-id-12',
+    recipientIdentifier: 'barry-harris-id',
+    responseDetails: {
+      minOptionsToBeRanked: 1,
+      maxOptionsToBeRanked: 2,
+      areDuplicatesAllowed: false,
+    } as FeedbackRankRecipientsQuestionDetails,
+    isValid: true,
+  };
```

Comment:
I think 12 submission forms data is a bit too much, maybe we can use 3

Reply:
I also worry that it's too much clutter, but cutting down on response submissions might cause us to lose out on completeness on testing all the types of questions. Would testing a handful of them rather than all be sufficient?